torq.q

```diff
@@ -553,6 +553,13 @@ reloadnamecode:{
 	loadspeccode["/",string procname]'[`KDBCODE`KDBSERVCODE`KDBAPPCODE];
 	};
 
+// execute system commands
+sys:{[cmd]
+	.lg.o[`system;"executing system command: ",cmd];
+	catcherror:{[cmd;error] .lg.e[`system;"failed to execute ",cmd,": ",error];error};
```

Comment:
These needs to re-throw the error rather than return a string representing the error i.e. 'error